build.assets/grpc/Dockerfile

```diff
@@ -12,26 +12,21 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM golang:1.10-stretch
+FROM golang:1.14-stretch
```

Comment:
Out of curiosity: why not jump to 1.15 as long as we're taking the risk of a bump?

Reply:
I could, just didn't want to add a lot of spinning wheels into already significant changes in PR.